### PR TITLE
update readme

### PR DESCRIPTION
**Pull request by javabster**
closes https://github.com/Qiskit/documentation/issues/12
closes https://github.com/Qiskit/documentation/issues/14

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,10 +1,60 @@
 # Qiskit docs
 
-The documentation content home for https://docs.quantum-computing.ibm.com. Refer to https://github.com/Qiskit/qiskit to make changes to the docs at https://qiskit.org/documentation.
+The documentation content home for https://docs.quantum-computing.ibm.com. Note this repo does not contain content for https://learning.quantum-computing.ibm.com/ or https://qiskit.org. Refer to https://github.com/Qiskit/qiskit to make changes to the docs at https://qiskit.org/documentation.
 
-# How tos
 
-## Pre-requisites to run tools locally
+# Improving IBM Quantum & Qiskit Documentation
+Maintaining up to date documentation is a huge challenge for any software project, especially in a field like quantum computing where the pace at which new research and technological capabilities get released is incredibly fast. As a result, we greatly appreciate any who take the time to support us in keeping this content accurate and up to the highest quality standard possible to benefit the broadest range of users.
```
Comment (abbycross) [inline README.md:7]:
```suggestion
Maintaining up-to-date documentation is a huge challenge for any software project, especially in a field like quantum computing where the pace at which advances in new research and technological capabilities happen incredibly fast. As a result, we greatly appreciate any who take the time to support us in keeping this content accurate and up to the highest quality standard possible to benefit the broadest range of users.
```

File: README.md [hunk 2]
```diff
@@ -1,10 +1,60 @@
...
+
+Read on for more information about how to support this project:
+
+## Best ways to contribute to Documentation
+### 1. Report bugs, inaccuracies or general content issues
+
+This is the quickest, easiest and most helpful way to contribute to this project and improve the quality of Qiskit and IBM Quantum documentation. There are a few different ways to report issues depending on where it was found:
```
Comment (abbycross) [inline README.md:14]:
```suggestion
This is the quickest, easiest, and most helpful way to contribute to this project and improve the quality of Qiskit and IBM Quantum documentation. There are a few different ways to report issues, depending on where it was found:
```

File: README.md [hunk 3]
```diff
@@ -1,10 +1,60 @@
...
+
+- For problems you've found in the [Qiskit API Reference](https://docs.quantum-computing.ibm.com/api/qiskit) section, open an issue in the Qiskit repo [here](https://github.com/Qiskit/qiskit/issues/new/choose)
```
Comment (abbycross) [inline README.md:16]:
```suggestion
- For problems you've found in the [Qiskit API Reference](https://docs.quantum-computing.ibm.com/api/qiskit) section, open an issue in the Qiskit repo [here](https://github.com/Qiskit/qiskit/issues/new/choose).
```

File: README.md [hunk 4]
```diff
@@ -1,10 +1,60 @@
...
+- For problems you've found in the [Qiskit Runtime IBM Client](https://docs.quantum-computing.ibm.com/api/qiskit-ibm-runtime/runtime_service) section, open an issue in the Qiskit IBM Runtime repo [here](https://github.com/Qiskit/qiskit-ibm-runtime/issues/new/choose)
```
Comment (abbycross) [inline README.md:17]:
```suggestion
- For problems you've found in the [Qiskit Runtime IBM Client](https://docs.quantum-computing.ibm.com/api/qiskit-ibm-runtime/runtime_service) section, open an issue in the Qiskit IBM Runtime repo [here](https://github.com/Qiskit/qiskit-ibm-runtime/issues/new/choose).
```

File: README.md [hunk 5]
```diff
@@ -1,10 +1,60 @@
...
+- For problems you've found in any other section of [docs](https://docs.quantum-computing.ibm.com), open a content bug issue [here](https://github.com/Qiskit/documentation/issues/new/choose).
+
+### 2. Suggest new content
+
+If you think there are gaps in our documentation, or sections that could be expanded upon, we invite you to open a new content request issue [here](https://github.com/Qiskit/documentation/issues/new/choose). 
+
+Not every new content suggestion is a good fit for docs, nor are we able to prioritise every request immediately. However, we will do our best to respond to content requests in a timely manner, and we greatly appreicate our community's efforts in generating new ideas.
```
Comment (abbycross) [inline README.md:24]:
```suggestion
Not every new content suggestion is a good fit for docs, nor are we able to prioritize every request immediately. However, we will do our best to respond to content requests in a timely manner, and we greatly appreciate our community's efforts in generating new ideas.
```

File: README.md [hunk 6]
```diff
@@ -1,10 +1,60 @@
...
+
+If you are interested in writing the new content yourself, or already have some draft work you think could be integrated, please also mention that in the issue description. If your content suggestion is accepted we will let you know and work with you to get the content written and reviewed.
```
Comment (abbycross) [inline README.md:26]:
```suggestion
If you are interested in writing the new content yourself, or already have some draft work you think could be integrated, please also mention that in the issue description. If your content suggestion is accepted, we will let you know and work with you to get the content written and reviewed.
```

File: README.md [hunk 7]
```diff
@@ -1,10 +1,60 @@
...
+
+Please note: we DO NOT accept unsolicited PRs for new pages or large updates to existing content. The content that we include in docs is carefully planned and curated by our content team and must go through the appropriate review process to ensure the quality is of the highest possible standard before deploying to production. As a result we are very selective with which content suggestions are approved and it is unlikely that PRs submitted without an associated approved content request will be accepted.
```
Comment (abbycross) [inline README.md:28]:
```suggestion
Please note: we DO NOT accept unsolicited PRs for new pages or large updates to existing content. The content that we include in docs is carefully planned and curated by our content team and must go through the appropriate review process to ensure the quality is of the highest possible standard before deploying to production. As a result we are very selective with which content suggestions are approved, and it is unlikely that PRs submitted without an associated approved content request will be accepted.
```

File: README.md [hunk 8]
```diff
@@ -1,10 +1,60 @@
...
+
+### 3. Validate exisiting issues
```
Comment (abbycross) [inline README.md:30]:
```suggestion
### 3. Validate existing issues
```

File: README.md [hunk 9]
```diff
@@ -1,10 +1,60 @@
...
+
+You can help the maintainer team prioritise already open issues by doing the following:
```
Comment (abbycross) [inline README.md:32]:
```suggestion
You can help the team prioritize already-open issues by doing the following:
```

File: README.md [hunk 10]
```diff
@@ -1,10 +1,60 @@
...
+
+- For bug reports, leave a comment in the issue if you have also been experiencing the same problem and can reproduce it (include as much information as you can, e.g.  browser type, qiskit version etc.)
```
Comment (abbycross) [inline README.md:34]:
```suggestion
- For bug reports, leave a comment in the issue if you have also been experiencing the same problem and can reproduce it (include as much information as you can, e.g.,  browser type, Qiskit version, etc.).
```

File: README.md [hunk 11]
```diff
@@ -1,10 +1,60 @@
...
+- For new content requests, leave a comment or upvote (👍) in the issue if you also would like to see that new content added.
+
+### 4. Fix an open issue
+
+You can look through the open issues we have in this repo and address them with a PR. We recommend focusing on issues with the "good first issue" or "help wanted" label. 
+
+Before getting started on an issue remember to do the following:
```
Comment (abbycross) [inline README.md:41]:
```suggestion
Before getting started on an issue, remember to do the following:
```

File: README.md [hunk 12]
```diff
@@ -1,10 +1,60 @@
...
+
+1. Read the Code of Conduct (TODO add link)
+2. Check for issues with the "good first issue" or "help wanted" label
+3. Select an issue that is not already assigned to someone and leave a comment to request to be assigned
+
+Once you have an issue to work on see the "How to work with this repo" section below to get going, then open a PR.
```
Comment (abbycross) [inline README.md:47]:
```suggestion
Once you have an issue to work on, see the "How to work with this repo" section below to get going, then open a PR.
```

File: README.md [hunk 13]
```diff
@@ -1,10 +1,60 @@
...
+
+Before opening a PR remember to do the following:
```
Comment (abbycross) [inline README.md:49]:
```suggestion
Before opening a PR, remember to do the following:
```

File: README.md [hunk 14]
```diff
@@ -1,10 +1,60 @@
...
+1. Check that you have addressed all the requirements from the original issue
+2. Run the spell checker
+3. Use the GitHub "fixes" notation to [link your PR to the issue](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue) you are addressing
+
+
+# How to work with this repo
+
+## Pre-requisites to building the docs locally
```
Comment (abbycross) [inline README.md:57]:
```suggestion
## Prerequisites to building the docs locally
```

**Review by abbycross — COMMENTED** [review]
Really tiny tweaks, mostly commas

File: README.md [hunk 15]
```diff
@@ -1,10 +1,60 @@
...
+- For bug reports, leave a comment in the issue if you have also been experiencing the same problem and can reproduce it (include as much information as you can, e.g.  browser type, qiskit version etc.)
+- For new content requests, leave a comment or upvote (👍) in the issue if you also would like to see that new content added.
+
+### 4. Fix an open issue
+
+You can look through the open issues we have in this repo and address them with a PR. We recommend focusing on issues with the "good first issue" or "help wanted" label. 
+
+Before getting started on an issue remember to do the following:
+
+1. Read the Code of Conduct (TODO add link)
```
Comment (Eric-Arellano) [inline README.md:43]: Should we link to https://github.com/Qiskit/qiskit/blob/main/CODE_OF_CONDUCT.md for now?

File: README.md [hunk 16]
```diff
@@ -1,10 +1,60 @@
...
+2. Check for issues with the "good first issue" or "help wanted" label
```
Comment (Eric-Arellano) [inline README.md:44]: We don't have these labels. If we want them, maybe consolidate around one?

**Review by Eric-Arellano — COMMENTED** [review]
Thanks, Abby!

File: README.md (hunk 16 above)
Comment (abbycross) [reply README.md:44]: I added a "good first issue" label just now.

Comment (Eric-Arellano) [reply README.md:44]: Thanks! I personally like "good first issue" more than "help wanted". I think it makes sense to have only one. They serve such a similar purpose.

File: README.md (hunk 5 above)
Comment (javabster) [reply README.md:24]: 😑😑😑😑

File: README.md (hunk 8 above)
Comment (javabster) [reply README.md:30]: damn you can really tell I wrote this page in a rush can't you lol

File: README.md (hunk 9 above)
Comment (javabster) [reply README.md:32]: Is there a reason you don't like the term "maintainer"? I'm not strongly attached to it, just curious

File: README.md (hunk 16 above)
Comment (javabster) [reply README.md:44]: we've used both pretty effectively in the [qiskit repo](https://github.com/Qiskit/qiskit/blob/main/CONTRIBUTING.md#choose-an-issue-to-work-on) in the past, but happy to keep it just to GFI for now and we can update later if we need

**Comment by javabster** [comment]
Thanks for the comments all, this is ready for re-review :)

File: CODE_OF_CONDUCT.md [hunk 17]
```diff
@@ -0,0 +1,9 @@
+<!-- Copyright Contributors to the Qiskit project. -->
+
+# Code of Conduct
+All members of this project agree to adhere to the Qiskit Code of Conduct listed at [https://github.com/Qiskit/qiskit/blob/master/CODE_OF_CONDUCT.md](https://github.com/Qiskit/qiskit/blob/master/CODE_OF_CONDUCT.md)
+
+----
+
+License: [CC BY 4.0](https://creativecommons.org/licenses/by/4.0/),
+Copyright Contributors to Qiskit.
```
Comment (Eric-Arellano) [inline CODE_OF_CONDUCT.md:9]: Idk why we have this? The repo is Apache license 2. More generally, I don't know why you'd have a license in Code of Conduct.

cc @1ucian0

**Review by Eric-Arellano — CHANGES_REQUESTED** [review]
Thanks, Abby! Blocking only because of the license question.

File: CODE_OF_CONDUCT.md (hunk 17 above)
Comment (1ucian0) [reply CODE_OF_CONDUCT.md:9]: It is not uncommon to have a license for the CoC (for exampe https://source.chromium.org/chromium/chromium/src/+/main:CODE_OF_CONDUCT.md).

With respect to the license, it is not a problem to have a file with a different license that is more permissive than the one in the repo (like in this case). Usual IANAL applies.

**Review by Eric-Arellano — APPROVED** [review]

**Pull request merged**